pkgs/development/tools/vulkan-validation-layers/default.nix

```diff
@@ -60,6 +60,16 @@ stdenv.mkDerivation rec {
     "-Wno-dev"
   ];
 
+  installPhase = ''
```

Comment:
If I understood correctly, you are rewriting the whole phase because it does not install certain files.

This being the case, you should put the extra installation under `postInstall` instead.

Reply:
Indeed. I did that, thank you for the suggestion.